src/components/pages/Login/index.tsx

```diff
@@ -29,12 +29,14 @@ const Login = () => {
     const [emailInput, setEmailInput] = useState("");
     const [passwordInput, setPasswordInput] = useState("");
     const [errorMsg, setErrorMsg] = useState("");
+    const [isActive, setIsActive] = useState(true)
```

Comment:
Maybe we could change the terminology from `isActive` to `isLoading`/`loading` since active doesn't really reflect the action. There is nothing to be active but it's definitely loading. 

```suggestion
    const [loading, setLoading] = useState(false)
```

Reply:
Same goes for Signup. 😃